python/lsst/cell_coadds/_stitched_aperture_correction.py

```diff
@@ -46,19 +51,34 @@ def __init__(self, ugrid: UniformGrid, gc: Mapping[Index2D, float]):
         self.ugrid = ugrid
         self.gc = gc
 
-    def evaluate(self, point: geom.Point2D | geom.Point2I) -> float:
+    def evaluate(
+        self, x: geom.Point2D | geom.Point2I | Iterable[float], y: Iterable[float] | None = None
+    ) -> float | np.ndarray:
```

Comment:
This is trying to mimic [this](https://github.com/lsst/meas_algorithms/blob/8180f54f504d26b75dddf5de960a59d9e249b88b/python/lsst/meas/algorithms/coaddBoundedField.py#L34) API.